go/hertz/main.go

```diff
@@ -12,6 +12,8 @@ func main() {
 		server.WithHostPorts(":3000"),
 		server.WithDisableHeaderNamesNormalizing(true),
 		server.WithDisablePrintRoute(true),
+		server.WithDisableDefaultDate(true),
+		server.WithDisableDefaultContentType(true),
```

Comment:
Not sure this one should be here (maybe it should be removed from fiber too)

Reply:
GET. 
If it is determined that alignment is not needed, I turn off PR.

Reply:
Feel free to remove this kind of option in all frameworks (including `fiber`